src/main/java/org/cbioportal/cdd/model/ClinicalAttributeMetadata.java

```diff
@@ -39,16 +41,22 @@
 })
 public class ClinicalAttributeMetadata {
 
+    @ApiModelProperty(value = "The column header")
     @JsonProperty("column_header")
     private String columnHeader;
+    @ApiModelProperty(value = "The name to be displayed in the cBio Portal")
     @JsonProperty("display_name")
     private String displayName;
+    @ApiModelProperty(value = "The description")
     @JsonProperty("description")
     private String description;
+    @ApiModelProperty(value = "The data type", allowableValues = "NUMBER,STRING")
```

Comment:
if we are describing how priorities changes display, we can also note that datatype can change what kind of graph is shown.